storybook/src/admin/form/file/FileSelectListItem.tsx

```diff
@@ -5,8 +5,15 @@ import { storiesOf } from "@storybook/react";
 import React from "react";
 
 storiesOf("@comet/admin/form/File", module)
+    // .addDecorator((story) => (
+    //     <Card sx={{ maxWidth: 440 }}>
+    //         <CardContent>
+    //             <Stack spacing={4}>{story()}</Stack>
+    //         </CardContent>
+    //     </Card>
+    // ))
```

Comment:
```suggestion
```

Why keep this code?

Reply:
Was just for debugging, committed it accidentally 😅